client/state/data-layer/wpcom/jetpack/settings/index.js

```diff
@@ -101,7 +110,7 @@ export const saveJetpackSettings = ( { dispatch }, action ) => {
 				path: '/jetpack-blogs/' + siteId + '/rest-api/',
 				body: {
 					path: '/jetpack/v4/settings/',
-					body: JSON.stringify( settings ),
+					body: JSON.stringify( toApi( settings ) ),
```

Comment:
Should the `JSON.stringify()` call be part of the `toApi()`?

Reply:
I think I've seen prior art where it wasn't. Not feeling to strongly about this.

Reply:
I'm fine either way too.